pod/perlfunc.pod

```diff
@@ -4621,33 +4627,110 @@ any code points over 0xFF.
 Opening in-memory files I<can> fail for a variety of reasons.  As with
 any other C<open>, check the return value for success.
 
-See L<perliol> for detailed info on PerlIO.
+I<Technical note>: This feature works only when Perl is built with
```

Comment:
As noted in the pending PR https://github.com/Perl/perl5/pull/17682, it actually hasn't been supported to build Perl without PerlIO since 5.16. This note is probably still useful but should be noted only for older perls.